config/routes.rb

```diff
@@ -3,170 +3,185 @@
     instance_eval(File.read(Rails.root.join('config/openstudios_routes.rb')))
   end
 
-  resources :media, only: %i[index show]
+  constraints subdomain: /^(www)?$/ do
```

Comment:
this should match `www` or empty string so `missionartists.org` or `www.missionartists.org`.  IRL the second is what it needs to match for production because we redirect *all* traffic to www with Nginx.